### PR TITLE
core: warning banner for PWA category deprecation

### PR DESCRIPTION
**Pull request by adrianaixba**
https://lighthouse-pwhq5k0h5-googlechrome.vercel.app/gh-pages/viewer/?gist=dfb70105a0eefd64866be4271423eaab

Screenshot 2024-01-11 at 1 02 41 PM

### CODE REVIEW

File: report/renderer/report-utils.js [hunk 1]
```diff
@@ -476,6 +476,9 @@ const UIStrings = {
   openInANewTabTooltip: 'Open in a new tab',
   /** Generic category name for all resources that could not be attributed to a 1st or 3rd party entity. */
   unattributable: 'Unattributable',
+
+  /** Message communicating the removal of the PWA category. */
+  pwaRemovalMessage: 'Lighthouse will be deprecating the PWA category with Lighthouse version 12.0. More information about its deprecation [here](https://developer.chrome.com/blog/update-install-criteria?hl=en). For recommended PWA testing, refer to the documentation [here](https://developer.chrome.com/docs/devtools/progressive-web-apps/).',
```
[the remark this answers is not in the document]
Comment (adrianaixba) [reply report/renderer/report-utils.js]: ah this was an artifact of copying/pasting.. i can remove the `?hl=en` :)

[the remark this answers is not in the document]
Comment (adamraine) [reply report/renderer/report-utils.js]: Looks like they don't cover every language we do. Let's just remove the `?hl=en`, maybe the site can automatically detect locale.

File: report/renderer/report-renderer.js [hunk 2]
```diff
@@ -284,6 +284,7 @@ export class ReportRenderer {
 
     const reportContainer = this._dom.createElement('div', 'lh-container');
     const reportSection = this._dom.createElement('div', 'lh-report');
+
```
Comment (adamraine) [inline report/renderer/report-renderer.js:287]: nit
```suggestion
```

File: report/renderer/report-utils.js [hunk 3]
```diff
@@ -476,6 +476,9 @@ const UIStrings = {
   openInANewTabTooltip: 'Open in a new tab',
   /** Generic category name for all resources that could not be attributed to a 1st or 3rd party entity. */
   unattributable: 'Unattributable',
+
+  /** Message communicating the removal of the PWA category. */
+  pwaRemovalMessage: 'Alongside [Chrome’s updated Installability Criteria](https://developer.chrome.com/blog/update-install-criteria), Lighthouse will be deprecating the PWA category in Lighthouse version 12.0. For recommended PWA testing, refer to the following [documentation](https://developer.chrome.com/docs/devtools/progressive-web-apps/).',
```
Comment (adamraine) [inline report/renderer/report-utils.js:481]: Slight reword WDYT?
```suggestion
  pwaRemovalMessage: 'Alongside [Chrome’s updated Installability Criteria](https://developer.chrome.com/blog/update-install-criteria), Lighthouse will be deprecating the PWA category in a future release. Please refer to the [updated PWA documentation](https://developer.chrome.com/docs/devtools/progressive-web-apps/) for future PWA testing.',
```

**Review by adamraine — APPROVED** [review]

**Pull request merged**